src/index.ts

```diff
@@ -63,6 +66,57 @@ export class MaticPOSClient extends SDKClient {
     }
     return this.posRootChainManager.exitERC20(txHash, options)
   }
+
+  networkAgnosticTransfer(childToken: address, recipientAddress: address, amount: BN, user: address | string) {
+    if (!this.web3Client.web3.utils.isAddress(this.web3Client.web3.utils.toChecksumAddress(childToken))) {
```

Comment:
No sense in converting to checksum address before checking if it is valid.

Reply:
Right, updated to check if it valid or not.